tigerblood.go

```diff
@@ -291,8 +291,10 @@ func (h *TigerbloodHandler) UpsertReputationByViolation(w http.ResponseWriter, r
 	var penalty, ok = h.violationPenalties[entry.Violation]
 	if !ok {
 		log.Printf("Could not find violation type: %s", entry.Violation)
-		penalty = 0
+		w.WriteHeader(http.StatusBadRequest)
+		w.Write([]byte("Violation type not found."))
```

Comment:
Shouldn't we `return` here? We're still going to insert to the database, which is a bit wasteful. I'm not sure if postgres will avoid performing the write, but it'd be better not to touch the database again at all.